### PR TITLE
Move `_scripts/` and `_configs/config.json` under `.github/`

### PR DESCRIPTION
**Pull request by eemeli**
Currently, it's not very clear which parts of the repo contents are manually maintained, and which are updated by automation. By moving the scripts used by automation to `.github/scripts/`, that's made a little more obvious. Moving `_configs/config.json` to `.github/update-config.json` also distinguishes it from the TOML configs under `_configs/` that are maintained by automation.

This PR should be merged at a time when no `update` PR is open.

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

**Comment by eemeli** [comment]
@bcolsson Please merge this PR after merging #71.

**Comment by bcolsson** [comment]
@eemeli - Thanks, I'll be pushing strings tomorrow morning and will push this after.

**Pull request merged**
